Domain/sdLitica.Entities/Analytics/ModulesOperations.cs

```diff
@@ -1,13 +1,15 @@
 ﻿using sdLitica.Entities.Abstractions;
 using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
 using System.Text;
 
 namespace sdLitica.Entities.Analytics
 {
     /// <summary>
     /// Linking entity to support many-to-many relationship for AnalyticsModule and AnalyticsOperation.
     /// </summary>
+    [Table("ANALYTICS_MODULES_OPERATIONS")]
     public class ModulesOperations : Entity
```

Comment:
It is easier to maintain the code when Tables and Entity Classes (from ORM) have the same name. It become simple to relate from one to another.

Reply:
I will rename entity class.

Reply:
+1